pkg/report/testdata/netbsd/report/12

```diff
@@ -0,0 +1,16 @@
+TITLE: ASan: Unauthorized Access in vioscsi_scsipi_request
```

Comment:
This was detected as corrupted before, right?

Reply:
Ah, I see, this is this one:
https://syzkaller.appspot.com/bug?extid=a0fcffa03e71ed75fe1f

Reply:
Yup

Reply:
Same also came as https://syzkaller.appspot.com/text?tag=CrashReport&x=17a538e7200000